update-bitwarden.sh

```diff
@@ -61,7 +61,7 @@ REBUILD_BB=${tmprebuild:-$REBUILD_BB}
 
 if [[ $REBUILD_BB =~ ^[Yy]$ ]]
 then
-    ./build.sh
+    $BITWARDEN_BASE/BitBetter/build.sh
```

Comment:
I think it should be:

```suggestion
    $SCRIPT_BASE/build.sh
```

Because not everyone installed BitBetter inside the Bitwarden base directory.

Reply:
Whatever works :)  I just know it wasn't running at all prior when called as a cron job.

Do you want to change it/take over the PR?  I'm not precious about it